manifests/i/Intel/OneAPI/BaseToolkit/2025.1.3.8/Intel.OneAPI.BaseToolkit.locale.en-US.yaml

```diff
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.4.6 $debug=NVS2.CRLF.7-5-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: Intel.OneAPI.BaseToolkit
+PackageVersion: 2025.1.3.8
+PackageLocale: en-US
+Publisher: Intel Corporation
+# PublisherUrl: https://www.intel.com/content/www/us/en/homepage.html
+PublisherSupportUrl: https://community.intel.com/
+# PrivacyUrl: https://www.intel.com/content/www/us/en/privacy/intel-privacy-notice.html
+# Author:
+PackageName: Intel® oneAPI Base Toolkit
+# PackageUrl: https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html
+License: Proprietary
+# LicenseUrl:
+Copyright: © Intel Corporation
+# CopyrightUrl: https://www.intel.com/content/www/us/en/legal/terms-of-use.html
+ShortDescription: Heterogeneous Development across CPUs and GPUs
+Description: |-
+  The Intel® oneAPI Base Toolkit (Base Kit) is a core set of tools and libraries
+  for developing high-performance, data-centric applications across diverse
+  architectures. It features an industry-leading C++ compiler that implements
+  SYCL*, an evolution of C++ for heterogeneous computing.
+
+  Domain-specific libraries provide drop-in acceleration across relevant
+  architectures. Enhanced profiling, design assistance, and debug tools complete
+  the kit.
+# Moniker:
+# Tags:
+# ReleaseNotes:
+# ReleaseNotesUrl: https://www.intel.com/content/www/us/en/developer/articles/release-notes/oneapi-base-toolkit/2025.html
+# PurchaseUrl:
+# InstallationNotes:
```

Comment:
```suggestion
InstallationNotes: |-
  The default installation pass following switches to the installer:
  `-a --silent --eula accept -p=NEED_VS2019_INTEGRATION=0 -p=NEED_VS2022_INTEGRATION=0`
  It WILL NOT integrate this toolkit to VSIDE 2019/2022. If you need this, please
  reinstall this package with `--override` switch and override `NEED_VSXXXX_INTEGRATION`
  as value 1.
```